komga-webui/src/components/dialogs/EditSeriesDialog.vue

```diff
@@ -311,6 +315,32 @@
               </v-container>
             </v-card>
           </v-tab-item>
+
+          <!--  Tab: Thumbnails  -->
+          <v-tab-item v-if="single">
+            <v-card flat>
+              <v-container fluid>
+                <!-- Upload -->
+                <v-row>
+                  <drop-zone @onInputChange="inputChangeHandler" />
+                </v-row>
+
+                <!-- Gallery -->
+                <v-container class="d-flex flex-row flex-wrap">
```

Comment:
whenever a `v-container` is used, it should always also has a `v-row` and a `v-col`